src/embed.ts

```diff
@@ -1,13 +1,15 @@
 import { FastPath, Doc, doc, ParserOptions } from 'prettier';
 import { PrintFn } from './print';
-import { Node, AttributeNode, TextNode } from './print/nodes';
+import { Node, AttributeNode, TextNode, ElementNode } from './print/nodes';
 import { getText } from './lib/getText';
 
 const {
     builders: { concat, hardline, group, indent },
     utils: { removeLines },
 } = doc;
 
+const supportedLanguages = ['ts', 'js', 'css', 'scss'];
```

Comment:
i'm not too certain of either what languages we really support or for that matter what names the languages are supposed to have in the tag. Is it `ts` or `typescript`? Or is either ok?

Reply:
You can do either `lang="<language>"` or `type="text/<language>"`. For TS/JS, `ts`/`typescript` and `js`/`javascript` work. `less` and `postcss` (although this one might break, it will not most of the time, and so people will be annoyed) should be added to that list, too.

Reply:
Sounds like there could be more JavaScript-like and CSS-like languages that benefit from Prettier, that have a wide range of possible `type` or `lang` attributes? Maybe it would be easier to keep track of unsupported languages?

Reply:
That's a good idea!

Reply:
I think I found [the corresponding code](https://github.com/sveltejs/svelte-preprocess/blob/master/src/modules/language.ts#L52) in `svelte-preprocess`. It looks like in `lang` you can use any of `css`, `pcss`, `postcss`, `less`, `sass`, `scss`, `styl`, `stylus`, `js`, `pug`, `javascript`, `coffee`, `coffeescript`, `ts` or `typescript`.

Using `application/` or `text/` before seems only possible in the `type` attribute, presumably corresponding to MIME types. I can't find any examples of its use the in `svelte-preprocess` docs, though, so I think I'll ignore that attribute for now.

Ok, I'll switch from a whitelist to a blacklist. *Un*supported languages out of the ones above are CoffeeScript, Pug, Stylus.